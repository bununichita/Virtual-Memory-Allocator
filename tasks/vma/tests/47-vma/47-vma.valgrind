==36398== Memcheck, a memory error detector
==36398== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==36398== Using Valgrind-3.18.1 and LibVEX; rerun with -h for copyright info
==36398== Command: /home/nicky/Desktop/SDA/Virtual-Memory-Allocator/vma
==36398== Parent PID: 36397
==36398== 
==36398== 
==36398== HEAP SUMMARY:
==36398==     in use at exit: 0 bytes in 0 blocks
==36398==   total heap usage: 12 allocs, 12 frees, 8,392 bytes allocated
==36398== 
==36398== All heap blocks were freed -- no leaks are possible
==36398== 
==36398== For lists of detected and suppressed errors, rerun with: -s
==36398== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
