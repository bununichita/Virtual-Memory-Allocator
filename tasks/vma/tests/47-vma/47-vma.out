gigel
gigel
1
10
